src/macros.jl

```diff
@@ -120,6 +120,7 @@ struct TestItem
     setups::Vector{Symbol}
     retries::Int
     timeout::Union{Int,Nothing} # in seconds
+    skip::Union{Bool,Expr}
```

Comment:
Should this just be `skip::Any`?

e.g. what if it's `skip=SKIP_ORC_1` and then that's defined as a global variable before the runtests? (in which case the type would be `Symbol`.) Or maybe that wouldn't work anyway since we run this in a custom module...?

How about `skip=1`, which technically
```julia
julia> 1 == true
true
```

Reply:
i'm okay excluding `skip=1` and requiring a `Bool`. In general Julia requires a `Bool`for boolean things (like conditionals).